src/sonic-ctrmgrd/tests/ctrmgr_tools_test.py

```diff
@@ -0,0 +1,294 @@
+import os
+import pytest
+import sys
+
+from unittest.mock import MagicMock, patch
+
+from . import common_test
```

Comment:
`pytest` is a third-party lib, `unittest` is part of the standard library. Please re-group imports as follows:

```python3
import os
import sys
from unittest.mock import MagicMock, patch

import pytest

from . import common_test
```